eservice/pdo/sservice/block_store_manager.py

```diff
@@ -63,7 +63,7 @@ class BlockStoreManager(object) :
     symmetry with the storage service client.
     """
```

Comment:
A comment on this module, perhaps from earlier commits:

isn't the mark field of the metadata redundant? I guess, the garbage collection is one pass now, deleting entries as soon as they are found, instead of marking them first to delete later.

Reply:
no. the comment is correct. the blockstoremanager class has effectively the same interface as the storage service client so that we can use both local and remote instances in tests.

Reply:
Sorry if my comment was not clear. I was asking about the "mark" field of the metadata. This is not part of this PR, but I was asking anyway.